tests/gumjs/script.c

```diff
@@ -4256,6 +4257,15 @@ SCRIPT_TESTCASE (java_api_is_embedded)
   EXPECT_SEND_MESSAGE_WITH ("\"boolean\"");
 }
 
+SCRIPT_TESTCASE (native_pointer_loads_64_bits)
```

Comment:
Test should follow `native_pointer_to_match_pattern`, so the order matches. (This is not the case with all tests as I tried to order them from highest to lowest level so that the test-suite is more readable, while ordering them from lower to higher level in the list at the top. That is so fundamental breakage shows up in lower level tests first, where issues are easier to debug.)